oap-server/server-receiver-plugin/aws-firehose-receiver/README.MD

```diff
@@ -0,0 +1,7 @@
+### Notice
+The OTEL proto in this module copy from [opentelemetry-proto v0.7.0](https://github.com/open-telemetry/opentelemetry-proto/tree/v0.7.0) as AWS firehose only support OTEL 0.7, and rewrite follows :
```

Comment:
> AWS firehose only support OTEL 0.7

What does this mean? Does it mean the https://github.com/apache/skywalking/pull/10300 doesn't work at all or anything else?

Reply:
OTEL 0.7 has little difference from existing OTEL proto, for example:
```
instrumentationLibraryMetrics (0.7)-> scopeMetrics
doubleSummary, intSummary  (0.7) -> summary
labels (0.7) -> attributes
```

Reply:
We should reword, this receiver is using OTEL 0.7 protocol according to AWS doc.

And the copy should be mentioned in the root License file.

Reply:
Root License has already mentioned `opentelemetry-proto`, I have confirmed 0.7.0 is also `Apache 2.0`